assessment/collectors/project_analyzer/file_dependency_analyzer.go

```diff
@@ -157,7 +169,7 @@ func (g *GoDependencyAnalyzer) IsDAO(filePath string, fileContent string) bool {
 func (g *GoDependencyAnalyzer) GetExecutionOrder(projectDir string) (map[string]map[string]struct{}, [][]string) {
 	G := g.getDependencyGraph(projectDir)
 
-	sortedTasks, err := topologicalSort(G)
+	sortedTasks, err := g.TopologicalSort(G)
```

Comment:
it is not clear why topological sort is a property of the implementation of the graph